README.md

```diff
@@ -158,27 +158,40 @@ And define our draw() function:
                 serverUser: "neo4j",
                 serverPassword: "sorts-swims-burglaries",
             },
+            visConfig: {
+                nodes: {
+                    shape: 'dot'
+                },
+                edges: {
+                    arrows: {
+                        to: {enabled: true}
+                    }
+                },
+            },
             labels: {
                 Character: {
                     label: "name",
                     value: "pagerank",
                     group: "community",
                     [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
                         function: {
-                            title: (node) => viz.nodeToHtml(node, [
-                                "name",
-                                "pagerank"
-                            ])
+                            title: (node) => node.properties.name
                         }
                     }
                 }
             },
             relationships: {
                 INTERACTS: {
                     value: "weight"
+                    [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
+                        function: {
+                            title: (relationship) => relationship.properties.weight,
+                            label: (relationship) => 'INTERACTS',
```

Comment:
This is a good place to use static and not function 😅

Reply:
True :sweat_smile: